contracts/scripts/deploy.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * @notice Umbra Deployment script
+ * @dev At each contract deployment or transaction, we wait for the transaction to be mined before
+ * continuing to the next step. This is done because it makes it simpler to continue from that spot
+ * rather than restart the full deployment
+ * @dev To initialize a deploy:
+ *   `yarn deploy:umbra --network <network>`   (where network specifies a netwrk found in the hardhat.config.ts file)
+ * If deploying to a local node (--network localhost), first start Hardhat using `yarn hardhat node`
+ */
+const fs = require('fs');
+const hre = require('hardhat');
+const { ethers } = hre;
+
+const RINKEBY_PAYMASTER_PUBLIC_RELAYER_ADDRESS = '0x53C88539C65E0350408a2294C4A85eB3d8ce8789';
+
+const deployParams = require('./deployParams.json');
```

Comment:
For me this has a red squiggly line underneath that says

```
File 'umbra/contracts/scripts/deployParams.json' is not listed within the file list of project 'umbra/contracts/tsconfig.json'. Projects must list all files or use an 'include' pattern.ts(6307)
```

If you add `"scripts/**/*.json",` as an entry to `contracts/tsconfig.json` that should resolve this. For some reason the `"scripts/**/*"` doesn't cover this, even though you'd expect it to